api/references/activation-events.md

```diff
@@ -235,6 +256,139 @@ This activation event is emitted emitted whenever tasks of a certain type need t
 
 > **Note**: Beginning with VS Code 1.76.0, tasks contributed by your extension do not require a corresponding `onTaskType` activation event declaration for your extension to be activated.
 
+## onEditSession
+
+`onEditSession` is emitted whenever an edit session is accessed for the given scheme.
+
+```json
+...
+"activationEvents": [
+    "onEditSession:file"
+]
+...
+```
+
+## onSearch
```

Comment:
I think this would only be for a proposed API, and probably doesn't need to be documented here yet? cc @andreamah

Reply:
I guess that, since the provider APIs are currently trying to be finalized, we can keep this documentation in.